Sources/Logto/Core/LogtoCore+Fetch.swift

```diff
@@ -19,6 +19,7 @@ public extension LogtoCore {
         public let tokenEndpoint: String
         public let endSessionEndpoint: String
         public let revocationEndpoint: String
+        public let userinfoEndpoint: String
```

Comment:
Since we have `userInfo` in the file above, I think we should use `userInfoEndPoint` here as well

Reply:
LOG-1472

Reply:
OK, since OIDC uses `userinfo_endpoint`, so we will use `userinfoEndpoint` in this particular case only